esphome/components/i2c/i2c_bus_arduino.cpp

```diff
@@ -163,6 +163,17 @@ ErrorCode ArduinoI2CBus::write_readv(uint8_t address, const uint8_t *write_buffe
   }
 }
 
+ErrorCode ArduinoI2CBus::set_frequency(uint32_t frequency) {
+  if (this->frequency_ == frequency) {
+    return ERROR_OK;
+  }
+  this->frequency_ = frequency;
+  if (this->initialized_) {
+    wire_->setClock(frequency_);
```

Comment:
> All uses of class members and member functions should be prefixed with this-> to distinguish them from global functions/variables.

https://developers.esphome.io/contributing/code/#c-code-style

```suggestion
    this->wire_->setClock(this->frequency_);
```

Reply:
Fixed, and fixed several other places it was missed in the files I am already touching for this PR.